src/tools/remote/src/test/java/com/google/devtools/build/remote/worker/HttpCacheServerHandlerTest.java

```diff
@@ -0,0 +1,38 @@
+package com.google.devtools.build.remote.worker;
+
+import static org.junit.Assert.*;
+
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.JUnit4;
+
+@RunWith(JUnit4.class)
+public class HttpCacheServerHandlerTest {
+
+  private HttpCacheServerHandler handler = new HttpCacheServerHandler();
+
+  @Test
+  public void isUriValidWhenValidUri() {
+    assertTrue(handler.isUriValid("http://some-path.co.uk:8080/ac/1111111111111111111111111111111111111111111111111111111111111111"));
```

Comment:
Use an actual hash here. Maybe the one for the empty file `e3b0c44298fc1c149afbf4c8996fb92427ae41e4649b934ca495991b7852b855`